src/components/Personalization/createFetchDataHandler.js

```diff
@@ -69,15 +69,12 @@ export default ({
           [...pagePropositions, ...currentViewPropositions],
           nonRenderedPropositions
         ));
-        render()
-          .then(decisionsMeta => {
-            showContainers();
-            handleNotifications(decisionsMeta);
-          })
-          .catch(e => {
-            showContainers();
-            throw e;
-          });
+        render().then(handleNotifications);
```

Comment:
Should we keep the `catch` in here?

Reply:
The catch was unnecessary anyways because render shouldn't throw an exception. The rendering exceptions are caught and logged inside of render.